pkgs/development/python-modules/deltalake/default.nix

```diff
@@ -61,10 +62,14 @@ buildPythonPackage rec {
     pytestCheckHook
     pandas
     polars
+    deprecated
+    arro3-core
     pytest-benchmark
     pytest-cov-stub
     pytest-mock
     azure-storage-blob
+    pyarrow
+    pyarrow-hotfix
```

Comment:
pyarrow-hotfix is no longer necessary with pyarrow > 14.0.0 (https://nvd.nist.gov/vuln/detail/CVE-2023-47248).
```suggestion
```